spec/presenters/curation_concerns/file_set_presenter_spec.rb

```diff
@@ -77,4 +77,11 @@
       it { is_expected.to eq 'File' }
     end
   end
+
+  describe "single_use_links" do
+    it "returns ActiveRecord::Relation of all single use links for the file set" do
+      expect(SingleUseLink).to receive(:where).with(itemId: presenter.id)
```

Comment:
`itemId` is coming from the model https://github.com/projecthydra/curation_concerns/blob/master/app/models/single_use_link.rb#L8